securedrop/journalist.py

```diff
@@ -197,6 +197,8 @@ def admin_add_user():
                 else:
                     flash("An error occurred saving this user to the database",
```

Comment:
Can you update the flashed message too to encourage admins to check their logs? Something similar to what we use elsewhere should work. Nothing like pushing admins to report errors and help us improve exception handling.